src/main/java/org/jitsi/videobridge/Main.java

```diff
@@ -156,4 +103,14 @@ public static void main(String[] args)
             main.runMainProgramLoop(osgiBundles);
         }
     }
+
+    protected static void validateConfig()
+    {
+        ConfigValidator configValidator = new ConfigValidator("org.jitsi");
+        //TODO: pass command-line args as well
+        Set<String> configPropNames = JvbConfig.getConfig().withOnlyPath("videobridge").entrySet().stream()
```

Comment:
Is this equivalent to `.keySet()`?

Reply:
unfortunately this type doesn't define a `keySet()` here, so have to do it manually